ex4/r_kobayashi/main.py

```diff
@@ -0,0 +1,365 @@
+import argparse
+import os
+
+import librosa
+import numpy as np
+import matplotlib.pyplot as plt
+import scipy.signal as ssig
+
+import warnings
+
+warnings.filterwarnings("ignore")
+
+# 自己相関関数
+def autocorrelation(data, order=None):
+    """
+    parameter
+    ---
+    data:numpy.ndarray
+         audio time series
+    
+    returns
+    ---
+    ac:numpy.ndarray
+       autocorrelation
+    """
+    len_data = len(data)
+    ac = np.zeros(len_data)
+    if order == None:
+        order = len_data
+    for l in range(order):
+        for i in range(len_data - 1):
+            ac[i] += data[i] * data[i+1]

```

Comment:
ここ、1(イチ)じゃなくてl(エル)だと思います
```suggestion
        for i in range(len_data - l):
            ac[i] += data[i] * data[i+l]

```

Reply:
ac[i]に加算じゃなくて ac[l]に加算ですね（追加修正しておきました）